modules/building/pages/creating.adoc

```diff
@@ -91,6 +91,115 @@ For further details on PipelineRun permissions, please refer to the https://pipe
 
 . Once the PR is made, a build pipeline should start. You can track its progress in the {ProductName} UI or you can see the final status in GitHub after the pipeline completes. If the pipeline is successful, you can merge the PR.
 
+== Using different from default image repository
+
+When using ImageRepository object, you will end up with image repository in
+a default repository, if you want to use your own repository
+don't create ImageRepository and follow this section.
+
+Create your own repository in quay.io (or other registry, our example will be for quay.io)
+
+In quay.io web UI, go to your organization and click on *Create New Repository*, fill out repository name
+and visibility and click on *Create Public/Private Repository* button.
+eg. `quay.io/test_organization/own-repository`
+
+Create robot account in your organization and grant write permission to your repository,
+you can also create another robot account only with read permissions.
+
+In quay.io web UI, go to your organization, click on *Robot Accounts* in the left menu,
+then click on *Create Robot Account*, fill out name of robot account and click on *Create robot account* button,
+in the list of repos find your repository and set *write* permission for it and then click on *Add permissions* button.
+
+When you click on the newly created robot account on *Robot Accounts* page, you can get details of the account.
+In the openened popup for the account first you will see credentials *Username* and *Token* (we will use them later),
+when you click on *Kubernetes Secret* you can download secret yml file (we will use it later).
+
+Now we will have to create secret with repository credentials.
+
+=== Creating Secret via {ProductName} UI
+xref:building:creating-secrets.adoc#example-of-creating-a-quay-io-secret[create a secret]
+
+This will also add secret to service account `appstudio-pipeline` which will be used for build pipeline.
```

Comment:
```suggestion
You can now xref:building:creating-secrets.adoc#example-of-creating-a-quay-io-secret[create a secret] with the credentials above. Make sure to use the most narrowly valid scope for the `registry server address` field when adding the secret. When adding the secret with the UI, it will automatically be added to the `appstudio-pipeline` service account so that it can be leveraged by builds.
```

Reply:
updated